packages/shared/lib/core/wallet/stores/new-transaction-details.store.ts

```diff
@@ -0,0 +1,16 @@
+import { writable, Writable } from 'svelte/store'
+import { INewTransactionDetails } from '..'
+
+export const newTransactionDetails: Writable<INewTransactionDetails> = writable({})
```

Comment:
We typically initialize to `null` - we have to be careful initializing as an empty object since those are truthy whereas `null` is falsy.

Reply:
Initializing this to `null` throws errors when destructuring the object after it has been reset or `send Funds` button is clicked. Either we give it sensible default values or create an empty object and fill it in later and never expect this to be null. We never evaluate this object to be truthy or not.

Reply:
I think sensible default values make sense then, you could create a new `DEFAULT_NEW_TRANSACTION_DETAILS` constant :+1:

Reply:
The properties' values depend on what is shown on the `sendFormPopup` initially. All properties are optionally present, since we need compatibility with deep links, but also be able to start a blank transaction. A `DEFAULT_NEW_TRANSACTION_DETAILS` is an empty object. Do you want to make this explicit?